src/precompile.jl

```diff
@@ -1,3 +1,658 @@
 function _precompile_()
     ccall(:jl_generating_output, Cint, ()) == 1 || return nothing
+    precompile(Tuple{typeof(Plots.gr_set_gradient), Plots.Series})
+    precompile(Tuple{typeof(Plots.gr_display), Plots.Subplot{Plots.GRBackend}, Measures.Length{:mm, Float64}, Measures.Length{:mm, Float64}, Array{Float64, 1}})
```

Comment:
so there's no need to split gr_display?

Reply:
I still need to try that, but thought I'd put what I've got up.